content/en/docs/Creating derivatives/creating_bootable_images.md

```diff
@@ -55,6 +62,8 @@ In the example above, the cos-toolkit parts that are **required** are pulled in
 {{<package package="system/cloud-config" >}} is optional, but provides `cOS` defaults setting, like default user/password and so on. If you are not installing it directly, an equivalent cloud-config has to be provided in order to properly boot and run a system, see [oem configuration](../../customizing/oem_configuration).
 {{% /alert %}}
 
+The {{<package package="toolchain/cosign" >}} and {{<package package="toolchain/luet-cosign" >}} are optional packages that would install cosign and luet-cosign in order to verify the packages installed by luet in the next step. For more info, check the [cosign](../../getting-started/cosign) page.
```

Comment:
Maybe turn this into small paragraph about it ? feels like this information would get a bit lost

Reply:
well, its optional rigth? So I wanted to have it like the cloud-config package, but I can expand it a little bit

Reply:
updated!